vim25/types/types.go

```diff
@@ -18896,6 +18897,18 @@ type FileNotFound struct {
 	FileFault
 }
 
+func (f *FileNotFound) Error() string {
+	return fmt.Sprintf("fault: FileNotFound (file=%s)", f.File)
+}
+
+func (f *FileNotFound) Is(target error) bool {
```

Comment:
Implementing `Is()` seems required because `errors.Is()` internally checks for equality (even when implementing `Unwrap()`) . An empty/zero-value `target` error type is not equal to the provided `err` in `errors.Is(err, target)`.

Reply:
Unless we use `Errors.As()` which asserts the type